container/demo/README.md

```diff
@@ -0,0 +1,276 @@
+# GETTING STARTED WITH KS-NAV - A BASIC TUTORIAL
+
+## Table of Contents
+
+- [GETTING STARTED WITH KS-NAV - A BASIC TUTORIAL](#getting-started-with-ks-nav---a-basic-tutorial)
+  - [Table of Contents](#table-of-contents)
+  - [1 Overview](#1-overview)
+  - [2 A quick tutorial](#2-a-quick-tutorial)
+    - [2.1 Build Linux for ks-nav](#21-build-linux-for-ks-nav)
+    - [2.2 Run ks-nav on the built image](#22-run-ks-nav-on-the-built-image)
+  - [3 Detailed Tools Configurations](#3-detailed-tools-configurations)
+    - [3.1 Configuring kern\_bin\_db and the local database](#31-configuring-kern_bin_db-and-the-local-database)
+    - [3.2 Configuring nav and navweb for caller graph generation](#32-configuring-nav-and-navweb-for-caller-graph-generation)
+
+## 1 Overview
+
+To start using ks-nav, a Linux (or Yocto) build with debug symbols enabled is required. The tool then extracts symbols, functions and any useful data into a PostgreSQL database. 
+
+In this tutorial, you will:
+
+- Build a default branch from Linux - tag v6.6 is used in this tutorial
+- Extract symbols into a local database using ksnav-kdb
+- Run the navigator using ks-nav
+
+## 2 A quick tutorial
+
+### 2.1 Build Linux for ks-nav
+
+Skip this step if you already have a valid Linux build.
+The **demo/linux_app** folder includes a pre-configured Linux Configuration file and a Dockerfile.
+Use the commands below to create the container that will build the Linux image as defined in the configuration file **linux_config**.
+
+```bash 
+    # Move to the demo folder (should be the current directory: ks-nav/container/demo)
+    cd ./demo
+    # Optional step: Delete the ksbuild folder to start with a new environment
+    rm -rf ksbuild
+    # Create an artifacts folder
+    mkdir ksbuild
+    # Build Linux image with debug symbols enabled 
+    podman build -t linux-img -f Dockerfile_linux_app .
+    podman run -it --name linux-cont linux-img sh
+    podman cp linux-cont:/app $(pwd)/ksbuild/app
+``
+At the end of this step, the **./demo/ksbuild/app** folder should be populated with the Linux source and build artifacts. 
+
+### 2.2 Run ks-nav on the built image
+
+This step assumes that the **./demo/ksbuild/app** folder contains both Linux source and build artifacts.
+The artifacts generated from the Linux build in the previous step are now passed to another container - the ksnav container.
+
+```bash
+    # Move to the container folder (ks-nav/container)
+    cd container
+    # Create an output folder for ksnav images
+    mkdir ./demo/ksbuild/out_img
+    # Copy the ksnav config files to the /app folder 
+    cp ksnav_kdb_local.json ./demo/ksbuild/app/.
+    cp ksnav_nav_local.json ./demo/ksbuild/app/.
+```
+ks-nav can be run through a CLI, or through a web interface. We'll go through both options in this demo. 
+
+#### 2.2.1 Run ks-nav from CLI
+
```

Comment:
Talk about nav json and provide reference to the readme section (similar to json above)